peerpod-ctrl/go.mod

```diff
@@ -77,7 +77,7 @@ require (
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
-	github.com/kata-containers/kata-containers/src/runtime v0.0.0-20220913141151-9b49a6ddc6fd // indirect
+	github.com/kata-containers/kata-containers/src/runtime v0.0.0-20230606170044-338e18e4fd46 // indirect
```

Comment:
Could you revert this change, and run `go mod tidy` again?

We need a separated PR to update this line, since this `go.mod` file contains a reference to the `cloud-api-adaptor` main module as follows.

```
github.com/confidential-containers/cloud-api-adaptor v0.5.1-0.20230504043629-580abeb128aa
```

We need to update these lines at the same time in a separated PR after this PR gets merged.

Reply:
I'm slightly concerned that this means we will be building the release with some references that are 9 months old, but peerpod-ctrl isn't part of the release at the moment, so I'm happy to give this a go

Reply:
@yoheiueda - the linting tests still failed in the same way they did before I added the replace, so I think I need to revert your suggestion to get them passing in the short-term?